docs/interfaces/mx4/leds.md

```diff
@@ -1 +1,187 @@
-On the MX-4 platform the LEDs are configured under `/opt/hm/pic_attributes` 
+---
+title: LEDs (MX-4)
+tags:
+  - LED
+  - C61
+  - MX-4
+  - T30
+  - T30 FR
+  - CT
+---
+
+## Overview
+
+On the MX-4 platform the LEDs are configured under `/opt/hm/pic_attributes/LED` for complete list of [pic_attributes](pic_attributes.md)
+
+All leds except the power led can be controlled from linux see [Pre configured LED behavior](#Pre configured LED behavior).
+
+## WiFi LEDs
+
+The green wifi LED (mx4-wifi) is setup to be lit on netdev. you can overwrite this by set it to none or another trigger. Some platform does not have the mx4-wifi-red and this led is free to be used.
```

Comment:
```suggestion
The green WiFi LED (mx4-wifi) is set up to be lit on netdev. You can overwrite this by setting it to none or another trigger. Some platforms do not have the mx4-wifi-red and this led is free to be used.
```

Reply:
What do you mean by "this led is free to be used"

Reply:
there is no built in linux function that lit that led